### PR TITLE
Initital commit for plio

### PR DESCRIPTION
**Pull request by jlaura**
This is a Python only package that bundles up a number of I/O routines to support planetary science (e.g., Moon, Mars, Titan, etc.) data.

### CODE REVIEW

**Comment by jlaura** [comment]
@conda-forge/help-python I have updated the PR to be inline with conda-forge/staged-recipes master. It looks like the CI failures are occurring before getting to my PRed recipe additions. Can anyone advise if I have missed a configuration step?

File: recipes/plio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a189e7ee6973a209aee81ca412144599fdf7d2aefbb03c1d93c37212be053a7b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```
Comment (beckermr) [inline recipes/plio/meta.yaml]: This should be called `host` and should only contain packages needed to install the code, not run it. This is any package imported in the setup.py plus pip and python.

File: recipes/plio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - pandas
+    - sqlalchemy
+    - pyyaml
+    - affine
+    - networkx
+    - scipy
+    - certifi
+    - pyproj
+    - jinja2
+  run:
```
Comment (beckermr) [inline recipes/plio/meta.yaml]: You should list only direct dependencies of the package here. In other words, unless you are directly using protobuf or networkx, you can lose those deps. Listing deps of deps is bad practice.

File: recipes/plio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - affine
+    - networkx
+    - scipy
+    - certifi
+    - pyproj
+    - jinja2
+
+test:
+  imports:
+    - plio
```
Comment (beckermr) [inline recipes/plio/meta.yaml]: As a fellow astro person, the overlap with FITSIO PLIO image compression here is unfortunate! ;p

File: recipes/plio/meta.yaml (hunk 2 above)
Comment (jlaura) [reply recipes/plio/meta.yaml]: I went through the source and checked what was directly imported. Most of these were to handle different data representations (protobuf for a specification file, hdf5 for a h5py wrapper, network for adjacency information in a data set, etc.). I was though able to refactor out a section on the `plio` side and get the host down to just python and setup tools.

File: recipes/plio/meta.yaml (hunk 3 above)
Comment (jlaura) [reply recipes/plio/meta.yaml]: I had not idea! It is amazing the format separation between astronomy and surface planetary science. Maybe the new geoFITS will help some with that?

Comment (beckermr) [reply recipes/plio/meta.yaml]: Hahahah. Maybe!

**Comment by jlaura** [comment]
@beckermr I think that, once CI is back up, this is good to go? Perhaps tomorrow I should do an amend and force push to see if the CI is again working.

Thanks for taking look at this.

**Comment by beckermr** [comment]
Sounds good. It might be back up now too. Any commit will trigger.

**Comment by beckermr** [comment]
Yep. They need to confirm they are ok being listed. Once that is done we can merge!

**Comment by jlaura** [comment]
:+1: I am going to update the PR and remove him for now then given our current employment status. I can update with a PR once we open back up and I know he is checking gitter again.

**Comment by jlaura** [comment]
I should say, I can have @KelvinRR open a PR to add himself.  Thanks @beckermr

**Comment by beckermr** [comment]
Sounds great! To be clear, the pr to add a maintainer should be opened on the feedstock repo that will be created.

**Comment by jlaura** [comment]
:+1:

**Pull request merged**